### DISCUSSION TITLE
Filter Column

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5485] [General]
![image](https://user-images.githubusercontent.com/24252615/107963022-b1a55480-6fb8-11eb-8fc1-4db51524cfd2.png)
I want the addresses of the selected user to be listed
I searched but could not find full information how can I do this

**Comment by VictorTomaili** [comment]
https://serenity.is/docs/howto/how_to_setup_cascaded_editors

**Reply by havvanur160401046** [reply]
It worked, thank you very much
